Framework/DataHandling/test/SavePARTest.h

```diff
@@ -168,13 +168,6 @@ class SavePARTest : public CxxTest::TestSuite {
     loader.setPropertyValue("Workspace", input);
     loader.execute();
 
-    // mask the detector
-    Geometry::ParameterMap *m_Pmap = &(inputWS->instrumentParameters());
-    boost::shared_ptr<const Instrument> instru = inputWS->getInstrument();
-    Geometry::IDetector_const_sptr toMask = instru->getDetector(THEMASKED);
-    TS_ASSERT(toMask);
-    m_Pmap->addBool(toMask.get(), "masked", true);
-
```

Comment:
Well, on the other hand, if we leave it in, someone will assume that masking is being tested and works (which might be not true). So personally I would prefer better documentation (i.e., not putting misleading code there) over the offchance(?) that an entry in a map would break the algorithm in a detectable way.